src/types/types.ts

```diff
@@ -35,7 +35,11 @@ export type IMResponse = {
   nbPages?: number
 }
 
-export type SearchResponse = IStypes.SearchResponse & IMResponse
+export type SearchResponse = Omit<IStypes.SearchResponse, 'hits'> &
```

Comment:
Just for my knowledge, what does the `&` mean? (sorry not to have asked before, I'm sure it's obvious for you)

Reply:
It is an intersection between the types ([see doc for great examples](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces))